index.html

```diff
@@ -33,6 +33,13 @@
             COUNTRY: 'CZ',
             WORKING_HOURS_START: 8, // This has to correspond with the working hours set in opsgenie
             WORKING_HOURS_END: 16, // This has to correspond with the working hours set in opsgenie
+          },
+          ROAD_TO_PRODUCT: {
+            TITLE: 'Road to product',
```

Comment:
Title has to be exactly same (case) as name of your team in `On-call report` form "Select your team" dropdown. With that, team will be filled correctly. 
```suggestion
            TITLE: 'Road to Product', // This has to correspond with "Select your team" value from "On-call report"
```
<img width="693" alt="image" src="https://github.com/heureka/on-calls-pre-fill/assets/3725315/f83d5f63-73e0-45f0-8e0c-90cafa681bd7">